src/main/java/domain/Lotto.java

```diff
@@ -9,11 +9,15 @@
     public Lotto(PickNumberStrategy pickNumberStrategy) {
         this.lottoNumbers = new LottoNumbers(pickNumberStrategy);
     }
-
+    
     public LottoNumbers getNumbers() {
         return lottoNumbers;
     }
 
+    public boolean hasDuplicatedLottoNumber(LottoNumber lottoNumber) {
+        return lottoNumbers.getValues().contains(lottoNumber);
```

Comment:
```suggestion
        return lottoNumbers.contains(lottoNumber);
```

getter없이 기능을 제공해주면 좋겠네요 :)